### PR TITLE
Make sure we have a valid fileID for albums content and fix trashbin listener event

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ <skjnldsv@protonmail.com>

### CODE REVIEW

**Review by artonge — APPROVED** [review]
Which context can this happen?

File: src/views/AlbumContent.vue [hunk 1]
```diff
@@ -337,12 +337,13 @@ export default {
...
 
+				// Gen files info and filtering invalid files
 				const fetchedFiles = response.data
 					.map(file => genFileInfo(file))
+					.filter(file => file.fileid)
 
 				const fileIds = fetchedFiles
-					.map(file => file.fileid)
-					.map((fileId) => fileId.toString())
+					.map(file => file.fileid.toString())
```
Comment (artonge) [inline src/views/AlbumContent.vue:346]: That's wrong, the veritable will contain files and not file ids

**Review by artonge — CHANGES_REQUESTED** [review]

**Review by artonge — APPROVED** [review]

**Comment by skjnldsv** [comment]
> Which context can this happen?

File put to trashbin

**Review by artonge — APPROVED** [review]

**Pull request merged**